### PR TITLE
chore(package): switch to `@fabiosantoscode/uglify-es`

### PR DESCRIPTION
**Pull request by michael-ciniawsky**
### `Issues`

- #262

### `Notable Changes`

Switches from `uglify-es` (unmaintained)  to `@fabiosantoscode/uglify-es` (maintained)

### CODE REVIEW

File: src/uglify/versions.js [hunk 1]
```diff
@@ -1,5 +1,4 @@
 export default {
-  uglify: require('uglify-es/package.json').version, // eslint-disable-line global-require
+  uglify: require('@fabiosantoscode/uglify-es/package.json').version, // eslint-disable-line global-require
   plugin: require('../../package.json').version, // eslint-disable-line global-require
 };
```
Comment (evilebottnawi) [inline src/uglify/versions.js]: Lose newline in end of file?

File: src/uglify/versions.js [hunk 2]
```diff
@@ -1,5 +1,4 @@
...
-
```
Comment (evilebottnawi) [inline src/uglify/versions.js]: Here :smi

Comment (michael-ciniawsky) [reply src/uglify/versions.js]: mom, I didn't intentionally touch it (maybe `eslint --fix` on `precommit`)

Comment (evilebottnawi) [reply src/uglify/versions.js]: Very strange, let's return this newline

**Comment by evilebottnawi** [comment]
Close in favor https://github.com/webpack-contrib/uglifyjs-webpack-plugin/pull/293

**Pull request closed without merging**